addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -302,15 +291,21 @@ class Composer extends Component {
      * @param {Object} ev.detail
      * @param {string} ev.detail.unicode
      */
-    _onEmojiSelection(ev) {
-        this._textInputRef.comp.insertTextContent(ev.detail.unicode);
+    _insertEmojiInComposerTextInput(ev) {
+        this._textInputRef.comp.saveStateInStore();
+        this.storeDispatch(
+            'insertEmoji',
```

Comment:
suggestion: rename action to `insertEmojiInComposerTextInput` (the messaging action, not the handler function ;) )